gitlab/v4/objects.py

```diff
@@ -4042,6 +4062,8 @@ class Project(SaveMixin, ObjectDeleteMixin, RESTObject):
         ("variables", "ProjectVariableManager"),
         ("wikis", "ProjectWikiManager"),
         ("clusters", "ProjectClusterManager"),
+        ("additionalstatistics", "ProjectAdditionalStatisticsManager"),
+        ("issuesstatistics", "ProjectIssuesStatisticsManager"),
```

Comment:
I think `issue_statistics` would sound better.

Reply:
But above you have things like `protectedbranches`, not `protected_branches`?

Reply:
Also, GitLab API names it [issue**s** statistics](https://docs.gitlab.com/ee/api/issues_statistics.html#get-project-issues-statistics).